docs/installation-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Installation Guide"
+weight: 10
+---
+
+There are various ways you can use the Generator library to generate whaterver you want in your event driven architecture apps. Find your prefered method below:
+- [Using the AsyncAPI Generator CLI](#1-using-the-asyncapi-generator-cli)
+- [Use the Generator Library In Your Node.js Apps](#2-use-the-generator-library-in-your-nodejs-apps)
+  
+## Prerequisites
+Before you install and use the AsyncAPI CLI and the generator library, make sure you meet the prerequisites below then [install the CLI](#installation).
+1. Node.js v12.16 and higher
+2. Npm v6.13.7 and higher
+   
+To verify the versions of node and npm you have, run the following command on your terminal:
+```
+npm -v
+```
+```
+node -v
+```
+
+If you don't have either Node or Npm installed, do so using the [official node.js installer](https://nodejs.org/en/download/).
+
+If you have the right versions installed, proceed to the CLI installation guide below else, upgrade the version of npm or node if lower than the recommended versions specified above.
+
+## Using the AsyncAPI Generator CLI
```

Comment:
```suggestion
## Generator CLI
```

When writing **headers** in docs, focus on being **succinct**. Avoid unnecessary terms, verbs,  or articles _(i.e. the, of, etc)_. 

Example: `asyncapi.com/docs/tools/generator#generator-cli`.

Reply:
Gotcha! This makes so much sense, when writing I wasn't putting into consideration how the links to the different sections will look like or even SEO! Thank you for enlightening me